spec/features/dashboards/dashboards_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Dashboards' do
+  context 'admin' do
+    it 'renders' do
+      create_draws
+      log_in FactoryGirl.create(:admin)
+      expect(page).to have_content('Vesta')
```

Comment:
So I assume the plan is to either expand these or rework the whole thing when we release for real?